### PR TITLE
Asynchronous Code: Visualization Vids of Event Loop and Promises

### PR DESCRIPTION
**Pull request by inshal-haq**
## Because

Videos are extra helpful in further visualizing how JavaScript works in the background with the Browser.

## This PR

- Included in Assignment section because I felt it was that useful, but can decide if want to include them or not.
- Put them in position 4, 5 because I felt that learning order was best.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/asynchronous_javascript_and_apis/asynchronous_code.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ Of course, there are many more occasions where one would want to use Promises be
 1. Read this [Promises article](https://davidwalsh.name/promises). It's a good starting place and it's short and to the point.
 2. Watch this [Promises video](https://youtu.be/DHvZLI7Db8E).  It's a good place to get a feel for how one might actually use promises in the wild.
 3. Watch this [What is Event Loop? video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to understand how asynchronous code works in JavaScript.
-4. Read [promise basics](https://javascript.info/promise-basics), which is an excellent resource for beginners.
+4. Watch this [Event Loop - Visualized video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to further understand the Event Loop.
```
Comment (wise-king-sullyman) [inline javascript/asynchronous_javascript_and_apis/asynchronous_code.md:78]: It looks like this is a repeat of the previous video?

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Hello and thanks for making this PR! Sorry for the delay this got lost in my notifications.

One change request and this should be good.

File: javascript/asynchronous_javascript_and_apis/asynchronous_code.md (hunk 1 above)
Comment (inshal-haq) [reply javascript/asynchronous_javascript_and_apis/asynchronous_code.md:78]: Yeah they are just additional, repetitive videos. But thought the repetition and visualization were helpful in solidifying the knowledge. But yeah, can include in Additional resources instead, or if not necessary, then no need to include at all

Comment (wise-king-sullyman) [reply javascript/asynchronous_javascript_and_apis/asynchronous_code.md:78]: I think we might be talking past each other, I mean that it looks like your `Event Loop - Visualized` video is the same video that's linked under the `What is Event Loop?` link.

Comment (inshal-haq) [reply javascript/asynchronous_javascript_and_apis/asynchronous_code.md]: Ohh I see, sorry my mistake. Have included the updated link in my second commit.

4. Watch this [Event Loop - Visualized video](https://www.youtube.com/watch?v=eiC58R16hb8) to further understand the Event Loop

**Review by wise-king-sullyman — APPROVED** [review]
Resolved a couple merge conflicts, thanks!

Also sorry this got held up so long again, in the future if a maintainer requests changes on your PR please click the "request re-review" button once you've addressed the changes, makes it a bit easier for us to see when we need to take another look at a PR 🙂

**Pull request merged**